depmapomics/terra.py

```diff
@@ -166,26 +166,12 @@ def copyToWorkspace(
         "version",
         "datatype",
         "size",
-        "stripped_cell_line_name",
-        "participant_id",
```

Comment:
Yes terra requires participant ids. Since gumbo's seq table doesn't have a participant id column, loading.addSamplesToDepMapWorkspace adds it by mapping it from the model table before importing to terra. Is it important to keep a stripped_cell_line_name column in the workspace though? since we can look it up easily on gumbo

Reply:
so the goal in the future is to copy most of gumbo into terra because it is just hard to do back and forth to look for things otherwise.  (speaking from experience here)

Reply:
Makes sense. I just added another function to append information from the model table to the sequencing table. this way we can more easily have a table that resembles the old sample tracker